examples/cli-server.rs

```diff
@@ -0,0 +1,193 @@
+#[macro_use]
```

Comment:
Just noticed this... see my other comment about `server.rs` being the permanent thing.